unlock-app/src/__tests__/components/creator/CreatorAccount.test.js

```diff
@@ -12,8 +12,12 @@ describe('CreatorAccount', () => {
     const network = {
       name: 4,
     }
+    const conversion = {
+      USD: 195.99,
+    }
 
-    const wrapper = shallow(<CreatorAccount account={account} network={network} />)
+    const wrapper = shallow(<CreatorAccount account={account} network={network} conversion={conversion} />)
```

Comment:
I am not sure it makes a lot of sense to pass that `conversion` props here. It feels like it should probably only be loaded inside the `Balance` component?

Reply:
We think alike: I had previously said:

> Because I did it this way, we get the USD conversion anywhere Balance was used, in the Dashboard summary, lock creation form, and lock display, with minimal change to source. Most of the changes involve adding the currency conversion table to mapStateToProps and adding it as a prop to the dumb components. This could be refactored to change Balance to a smart component that retrieves the conversion table and reduce some code size. Up to you if you want me to do that prior to merge, I would recommend it, but I'm less familiar with the codebase.

:)

I'll make it so.

Reply:
done